Parsers/ASimRegistryEvent/Parsers/imRegistry.yaml

```diff
@@ -12,14 +12,22 @@ References:
   Link: https://aka.ms/ASimRegistryEventDoc
 - Title: ASIM
   Link: https:/aka.ms/AboutASIM
-
 Description: |
   This ASIM parser supports normalizing Registry Event logs from all supported sources to the ASIM Registry Event normalized schema.
 ParserName: imRegistry
+EquivalentBuiltInParser: _Im_RegistryEvent 
```

Comment:
Team, please add ASIM file for registry too.

Reply:
@vakohl OK Will add unifying ASIM file.
But do we need to add ASIM parser for SentinelOne Registry schema as well?

Reply:
pls remove the other non-existent parsers from ASIM file. Let me know incase you still get validation error